README.md

```diff
@@ -195,44 +220,79 @@ All the options are supported. The callback function also receives these exports
 
-#### Props
+#### ConsentManager Props
+
+This section lists the configurable props you can change when
+loading the out of the box Consent Manager. In [this demo](https://codepen.io/samuelkahr/pen/eYYrobB), these are all the props you can pass into the returned preferences object.
+
+<!-- TOC depthFrom:5 depthTo:5 withLinks:1 updateOnSave:1 orderedList:0 -->
+
+- [writeKey](#writekey)
+- [otherWriteKeys](#otherwritekeys)
+- [shouldRequireConsent](#shouldrequireconsent)
+- [closeBehavior](#closebehavior)
+- [implyConsentOnInteraction](#implyconsentoninteraction)
+- [defaultDestinationBehavior](#defaultdestinationbehavior)
+- [cookieDomain](#cookiedomain)
+- [bannerContent](#bannercontent)
+- [bannerSubContent](#bannersubcontent)
+- [bannerTextColor](#bannertextcolor)
+- [bannerBackgroundColor](#bannerbackgroundcolor)
+- [preferencesDialogTitle](#preferencesdialogtitle)
+- [preferencesDialogContent](#preferencesdialogcontent)
+- [cancelDialogTitle](#canceldialogtitle)
+- [cancelDialogContent](#canceldialogcontent)
+- [customCategories](#customcategories)
+-
+<!-- /TOC -->
+
 
 ##### writeKey
 
-Type: `string`
+**Type**: `string`
+**Default**: none
 
-The write key analytics.js should be loaded with.
+The Segment source-specific write key which you can find in your `analytics.load()` call in your Analytics.js snippet.
 
 ##### otherWriteKeys
 
-Type: `array<string>`<br>
-Default: `[]`
+**Type**: `array<string>`
+**Default**: `[]`
 
-Other write keys that you want to load destination information for. This is useful for including your server-side destinations in the consent manager, so that you can easily apply the user's tracking preferences to your server-side analytics too. _No data will be sent to these write keys._
+[Write keys](https://share.getcloudapp.com/jkunGzrw) for other sources so you can include destinations they use in the Consent Manager tool list. This only displays destinations that are not connected to the primary writeKey.
+
+The user’s consent preferences for these tools are not sent to these additional sources, but they are added to the `identify` call the Consent Manager sends to Segment. The user’s preferences can then be added to a raw data destination (like a data warehouse) or to a user profile in Personas.
 
 ##### shouldRequireConsent
 
-Type: `function`<br>
-Default: `() => true`
+**Type**: `function`
+**Default**: `() => true`
 
-Callback function that determines if consent is required before tracking can begin. Return `true` to show the consent banner and wait for consent (if no consent has been given yet). Return `false` to not show the consent banner and start tracking immediately (unless the user has opted out). The function can return a `Promise` that resolves to a boolean.
+Callback function that determines if consent is required before tracking begins. Set to `true` to show the consent banner, otherwise return `false` to not show consent banner.
```

Comment:
''otherwise return `false` to not show consent banner and start tracking immediately" maybe it is better since it gives a little more information.